src/graph/fixtures.ts

```diff
@@ -3,96 +3,152 @@ import {
   SavedItemStatus,
   Videoness,
 } from '../generated/graphql/types';
-import { ListItemObject, RestResponse } from './types';
+import { GraphSavedItemEdge, ListItemObject, RestResponse } from './types';
+export const seedDataRest: MockListItemObject = Object.freeze({
+  ids: ['1', '2'],
+  given_url: 'https://test.com',
+  given_title: 'title',
+  favorite: '1',
+  status: '0',
+  time_added: '1677818995',
+  time_updated: '1677818995',
+  time_read: '1677818995',
+  time_favorited: '1677818995',
+  resolved_title: 'title',
+  resolved_url: 'https://test.com',
+  title: 'title',
+  excerpt: 'excerpt',
+  is_article: '1',
+  is_index: '1',
+  has_video: '1',
+  has_image: '1',
+  word_count: '100',
+  lang: 'en',
+  time_to_read: 10,
+  amp_url: 'https://test.com',
+  top_image_url: 'https://test.com/image.jpg',
+});
 
-export const testSavedItemFragment = (id) => {
-  return {
-    id: id,
-    status: SavedItemStatus.Unread,
-    url: 'test.com',
-    isFavorite: true,
-    isArchived: true,
-    _createdAt: 1677818995,
-    _updatedAt: 1677818995,
-    favoritedAt: 1677818995,
-    archivedAt: 1677818995,
-    //can inject tagModel as parameter when mocking
-    tags: [
-      {
-        id: `tag-${id}-1`,
-        name: 'tag-name-1',
-      },
-      {
-        id: `tag-${id}-2`,
-        name: 'tag-name-2',
-      },
-    ],
-  };
+export type SavedItemFragment = Pick<
+  GraphSavedItemEdge['node'],
+  | '__typename'
+  | 'id'
+  | 'status'
+  | 'url'
+  | 'isFavorite'
+  | 'isArchived'
+  | '_createdAt'
+  | '_updatedAt'
+  | 'favoritedAt'
+  | 'archivedAt'
+>;
```

Comment:
Suggestion: I'm not totally aware of how we want this `SavedItemFragment` to look like but if it's fewer fields to omit from `GraphSavedItemEdge['node']` we can use that too.

Basically using `Omit<>` instead of `Pick<>` if the fields to omit are fewer than the ones to be picked.

Reply:
ah nice catch, done!